style.css

```diff
@@ -165,10 +166,21 @@ h6 {
 }
 .right-side-bar-container {
     grid-template-rows: var(--header-height) minmax(400px, auto) 400px;
+    max-height: 100vh;
+    overflow-x: auto;
+    max-height: 100vh;
+    overflow-y: scroll;
+    scrollbar-width: none; 
+    -ms-overflow-style: none;
+}
+.right-side-bar-container::-webkit-scrollbar { 
+    width: 0;
+    height: 0;
```

Comment:
allows the tweets to scroll, but keeps the other columns static. also hides the scrollbar